components/webui/imports/api/ingestion/server/publications.js

```diff
@@ -45,6 +64,94 @@ const refreshCompressionStats = async () => {
     await StatsCollection.updateAsync(filter, modifier, options);
 };
 
+/**
+ * Updates the compression jobs in the CompressionJobsCollection.
+ *
+ * @return {Promise<void>}
+ */
+const refreshCompressionJobs = async () => {
+    if (null !== compressionJobsRefreshTimeout) {
```

Comment:
Perhaps we can add a comment saying that we clear the timeout in case this method was not called due to the timeout expiring.